src/ResourceManager/CognitiveServices/Commands.Management.CognitiveServices/CognitiveServicesAccount/GetAzureCognitiveServicesAccountUsage.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.Azure.Commands.Management.CognitiveServices.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.CognitiveServices;
+using Microsoft.Azure.Management.CognitiveServices.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.CognitiveServices
+{
+    /// <summary>
+    /// Get Usages for Cognitive Services Account
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, CognitiveServicesAccountUsagesNounStr), OutputType(typeof(PSCognitiveServicesUsage))]
+    public class GetAzureCognitiveServicesAccountUsageCommand : CognitiveServicesAccountBaseCmdlet
+    {
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = "InputObjectParameterSet",
+            HelpMessage = "Cognitive Services Account Object.")]
+        [ValidateNotNullOrEmpty]
+        public PSCognitiveServicesAccount InputObject { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = "ResourceIdParameterSet",
+            HelpMessage = "Cognitive Services Account Resource ID.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
```

Comment:
You can remove ValueFromPipelineByPropertyName from ResourceGroupName and Name since piping scenarios are enabled.

Reply:
Removed.